jupyterlab-streaming/Dockerfile

```diff
@@ -61,11 +61,17 @@ RUN code-server --install-extension ms-python.python${VSCODE_PYTHON}
 COPY resources/vsc/settings.json /root/.local/share/code-server/Machine/
 COPY resources/vsc/.bashrc root/
 
-COPY requirements.txt .
-RUN pip install pip --upgrade \
-  && pip install --no-cache -r requirements.txt \
-  && pip cache purge \
-  && rm requirements.txt
+RUN pip install --no-cache-dir \
+    jupyterlab==$(conda list -f jupyterlab --json | grep -oP '(?<=\"version\": \")[^\"]*') \
+    jupyterlab-git \
+    jupyterlab-lsp \
+    jupyter-server-proxy 
```

Comment:
why have you decided to keep it in a Docker file instead of a requirements file? I would prefer requirements I think.

Reply:
@p-pekala I've uploaded back requirements and changed Dockerfile